dockers/docker-orchagent-cavm/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM docker-base
```

Comment:
I think we can reuse the docker-orchagent for this. We do not vendor specific orchagent docker. @stcheng will add start command to config copp rule. port_config_cavm.ini can be loaded based on the $onie_platform. see https://github.com/Azure/sonic-buildimage/blob/master/dockers/docker-orchagent/start.sh#L22